res/Sectors/M1105/M1105.xml

```diff
@@ -1833,7 +1833,7 @@ Left:   0041 0040 0039 0038 0037 0036 0035 0034 0033 0032 0031 0030 0029 0028 00
     <Y>3</Y>
     <X>-8</X>
     <Name Lang="as">Hkakhaeaw</Name>
-    <DataFile Source="CORE" Ref="http://www.ocean-of-storms.com/core/astrogat/i4.htm">hkakhaeaw.sec</DataFile>
+    <DataFile Source="CORE" Ref="http://www.ocean-of-storms.com/core/astrogat/i4.htm"> Milieu="M1105" Type="TabDelimited">hkakhaeaw.tab</DataFile>
```

Comment:
This renders the file invalid XML - make sure you inspect the `>` characters.

Also the `Milieu` attribute is unnecessary.